src/specBuilder/marks/markUtils.ts

```diff
@@ -77,10 +77,25 @@ export function getInteractive(children: MarkChildElement[]): boolean {
 /**
  * If a tooltip or popover exists on the mark, then set tooltip to true.
  */
-export function getTooltip(children: MarkChildElement[], name: string, nestedDatum?: boolean): SignalRef | undefined {
+export function getTooltip(
```

Comment:
This seems like a funky interface for getTooltip, but I see how it makes sense to toggle the booleans by name.  I like how python does this using `kwargs` but I don't think js has a comparable feature, besides this approach